zipline/finance/risk/period.py

```diff
@@ -208,38 +231,7 @@ def calculate_period_returns(self, returns):
         period_returns = (1. + returns).prod() - 1
         return period_returns
 
-    def calculate_volatility(self, daily_returns):
-        return np.std(daily_returns, ddof=1) * math.sqrt(self.num_trading_days)
-
-    def calculate_sharpe(self):
-        """
-        http://en.wikipedia.org/wiki/Sharpe_ratio
-        """
-        return sharpe_ratio(self.algorithm_volatility,
-                            self.algorithm_period_returns,
-                            self.treasury_period_return)
-
-    def calculate_sortino(self):
-        """
-        http://en.wikipedia.org/wiki/Sortino_ratio
-        """
-        mar = downside_risk(self.algorithm_returns,
-                            self.mean_algorithm_returns,
-                            self.num_trading_days)
-        # Hold on to downside risk for debugging purposes.
-        self.downside_risk = mar
-        return sortino_ratio(self.algorithm_period_returns,
-                             self.treasury_period_return,
-                             mar)
-
-    def calculate_information(self):
-        """
-        http://en.wikipedia.org/wiki/Information_ratio
-        """
-        return information_ratio(self.algorithm_returns,
-                                 self.benchmark_returns)
-
-    def calculate_beta(self):
+    def calculate_covariance(self):
```

Comment:
I believe that we were only calculating covariance as an input to pass into the beta calculation.
With `pyrisk` now handling the calculation, we may be able to remove the covariance calculation from Zipline completely.

(I checked the `to_dict` methods in `risk.cumulative` and `risk.period` and it does not appear there, meaning it is not exposed in risk packets.)